apps/voting/scripts/buidler-hooks.js

```diff
@@ -0,0 +1,61 @@
+let pct16
+
+let tokens, accounts
+
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+module.exports = {
+  postDao: async function ({ _experimentalAppInstaller, log }, bre) {
+    const bigExp = (x, y) =>
+      bre.web3.utils
```

Comment:
Alternative to this is just using `BN.js` or `web3-utils` directly, or moving this type of helper to the contract-helpers.